jenkins-builds/7.0.UP03-21.09-daint-gpu

```diff
@@ -70,7 +70,8 @@
  otf2_cli_profile-33c89b7-CrayGNU-21.09.eb              --set-default-module --installpath=/apps/${system}/UES/jenkins/7.0.UP03/21.09/${system}-gpu/tools
  pprof-1a94d86.eb                                       --set-default-module --installpath=/apps/${system}/UES/jenkins/7.0.UP03/21.09/${system}-gpu/tools
  Scalasca-2.6-CrayGNU-21.09.eb                          --set-default-module --installpath=/apps/${system}/UES/jenkins/7.0.UP03/21.09/${system}-gpu/tools
- Score-P-7.1-CrayGNU-21.09.eb                           --set-default-module --installpath=/apps/${system}/UES/jenkins/7.0.UP03/21.09/${system}-gpu/tools
+ Score-P-8.0-CrayGNU-21.09.eb                           --set-default-module --installpath=/apps/${system}/UES/jenkins/7.0.UP03/21.09/${system}-gpu/tools
+ Score-P-7.1-CrayGNU-21.09.eb                                                --installpath=/apps/${system}/UES/jenkins/7.0.UP03/21.09/${system}-gpu/tools
```

Comment:
```suggestion
 Score-P-7.1-CrayGNU-21.09.eb                            --set-default-module --installpath=/apps/${system}/UES/jenkins/7.0.UP03/21.09/${system}-gpu/tools
```

Reply:
I suggest to postpone installing `Score-P` release 8.0 until it is properly tested